website/docs/guides/custom-service-endpoints.html.md

```diff
@@ -239,25 +239,46 @@ provider "aws" {
   skip_requesting_account_id  = true
 
   endpoints {
-    apigateway     = "http://localhost:4567"
-    cloudformation = "http://localhost:4581"
-    cloudwatch     = "http://localhost:4582"
-    dynamodb       = "http://localhost:4569"
-    es             = "http://localhost:4578"
-    firehose       = "http://localhost:4573"
-    iam            = "http://localhost:4593"
-    kinesis        = "http://localhost:4568"
-    lambda         = "http://localhost:4574"
-    route53        = "http://localhost:4580"
-    redshift       = "http://localhost:4577"
-    s3             = "http://localhost:4572"
-    secretsmanager = "http://localhost:4584"
-    ses            = "http://localhost:4579"
-    sns            = "http://localhost:4575"
-    sqs            = "http://localhost:4576"
-    ssm            = "http://localhost:4583"
-    stepfunctions  = "http://localhost:4585"
-    sts            = "http://localhost:4592"
+    apigateway     = "http://localhost:4566"
+    cloudformation = "http://localhost:4566"
+    cloudwatch     = "http://localhost:4566"
+    dynamodb       = "http://localhost:4566"
+    es             = "http://localhost:4566"
+    firehose       = "http://localhost:4566"
+    iam            = "http://localhost:4566"
+    kinesis        = "http://localhost:4566"
+    lambda         = "http://localhost:4566"
+    route53        = "http://localhost:4566"
+    redshift       = "http://localhost:4566"
+    s3             = "http://localhost:4566"
+    secretsmanager = "http://localhost:4566"
+    ses            = "http://localhost:4566"
+    sns            = "http://localhost:4566"
+    sqs            = "http://localhost:4566"
+    ssm            = "http://localhost:4566"
+    stepfunctions  = "http://localhost:4566"
+    sts            = "http://localhost:4566"
+  }
+}
+```
+
+#### S3 Backend
```

Comment:
Hi @artamonovkirill 👋 Please note that the Terraform S3 Backend, while it shares some authentication logic with the Terraform AWS Provider for consistency, is wholly separate and documented here: https://github.com/hashicorp/terraform/blob/master/website/docs/backends/types/s3.html.md

We would prefer not to conflate these separate pieces of functionality in the Terraform AWS Provider documentation since any Terraform backend can be chosen. We could instead potentially link to the relevant (updated) page here in the future. 👍

Reply:
@bflad , thanks for the explanation.
I'll create a PR for the backend documentation.
Is it ok if I leave this PR open and update it once backend documentation is available?

Reply:
Closing this PR, as my PR to terraform main repo was rejected